src/components/SitewideBanner/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import Cookies from 'js-cookie';
+import {
+  Alert, Container,
+} from 'react-bootstrap';
+
+const SitewideBanner = ({
+  message, type, dismissible, cookieName, cookieExpiryDays,
+}) => {
+  const [isVisible, setIsVisible] = useState(true);
+
+  useEffect(() => {
+    if (cookieName && Cookies.get(cookieName)) {
+      setIsVisible(false);
+    }
+  }, [cookieName]);
+
+  const handleDismiss = () => {
+    setIsVisible(false);
+    if (cookieName) {
+      Cookies.set(cookieName, 'true', { expires: cookieExpiryDays });
+    }
+  };
+
+  if (!isVisible) {
+    return null;
+  }
```

Comment:
nit: instead of returning null here, we can use if-else format that has been used in other places in MFEs in opened. It makes the code consistent. This code looks a bit out of place.